e2etests/cloud/interop/main_test.go

```diff
@@ -27,10 +26,6 @@ func TestMain(m *testing.M) {
 		fmt.Printf("TMC_API_HOST conflicts with TMC_API_URL")
 		os.Exit(1)
 	}
-	if apiHost == cloud.Host || apiURL == cloud.BaseURL {
```

Comment:
I removed this because in a separate PR I will create another workflow to run this against our `terramate-tests` prod org.

Reply:
The workflow uses `TMC_API_HOST: api.stg.terramate.io` though and not prod, is this intended?

Reply:
yes, @sixstone-qq asked for running on staging.